### ISSUE TITLE
Higher input image resolution

### ISSUE THREAD

**Issue by mayank-g20** [experiencor/keras-yolo2#317]
How do I fix the image size issue? I have an image of shape (1300, 1300, 8) with annotations available in JSON, I have prepared the text file required for the annotations but how do I pass a higher dimension image? And it is uint16.

Any help on this issue or next steps?

**Comment by rodrigo2019** [comment]
I think it is not so easy, because in this repo use functions from opencv and imgaug that probably does't work with 8 channels images

**Comment by mayank-g20** [comment]
Yeah, that has been one of the major issues with my project as well. I have to split my images into 4 channels, work on data augmentation and join them again later. Do you think there can be a workaround that? In the meanwhile, I will just select the RGB channels and see how the model performs.

However, any ideas or hints on how to manage the large matrix size? (1300 by 1300), should I alter the code or just create patches of 416 by 416 and feed those patched in? 

Though creating those patches out of 1300,1300 images would be a challenge itself.

P.S: I am trying to apply this algorithm to the SpaceNet data set.

**Comment by NPetsky** [comment]
I think it is no problem to train on 1300x1300 images but of course you have to calculate new anchors for your dataset and change the input_size in config.json
